skl2onnx/operator_converters/k_bins_discretiser.py

```diff
@@ -23,13 +26,7 @@ def convert_sklearn_k_bins_discretiser(scope, operator, container):
     ranges = list(map(lambda e: e[1:-1] if len(e) > 2
                       else [np.finfo(np.float32).max], op.bin_edges_))
     digitised_output_name = [None] * len(ranges)
-    instances = operator.inputs[0].type.shape[0]
-
-    last_column_name = scope.get_unique_variable_name('last_column')
-
-    container.add_initializer(
-        last_column_name, onnx_proto.TensorProto.FLOAT,
-        [instances, 1], np.ones(instances))
+    last_column_name = None
```

Comment:
Can you explain this change? You are setting last_column_name to None and then checking if last_column_name is None. I don't see any updates in between.